pcf/particle/aws/ecr/ecr_repository.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2018 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pcf.core import State
+from pcf.util import pcf_util
+from pcf.core.aws_resource import AWSResource
+from pcf.core.pcf_exceptions import *
```

Comment:
import exceptions you need

Reply:
Changed to import only the exception that's used